examples/plot_fmri_data_example.py

```diff
@@ -152,18 +153,23 @@ def preprocess_haxby(subject=2, memory=None):
 SVR_permutation_test_inference = False
 if SVR_permutation_test_inference:
     # We computed the regularization parameter by CV (C = 0.1)
-    pval_corr_svr_perm_test, one_minus_pval_corr_svr_perm_test = permutation_test_cv(
-        X, y, n_permutations=50, C=0.1
+    estimator = LinearSVR(C=0.1)
+    weight_svr, weight_svr_distribution = permutation_test(X, y, n_permutations=50)
+    pval_corr_svr_perm_test, one_minus_pval_corr_svr_perm_test = permutation_test_pval(
+        weight_svr, weight_svr_distribution
     )
 
 # Another method is to compute the p-values by permutation test from the
 # Ridge decoder. The solution provided by this method should be very close to
 # the previous one and the computation time is much shorter: around 20 seconds.
 
 estimator = Ridge()
```

Comment:
Same here, I would suggest highlighting that CV can be used
```suggestion
estimator = RidgeCV(alphas=np.logspace(-3, 3, 10))
```

Reply:
I think this can be done in other Pull request.
What do you think?

Reply:
I didn't see that and thought this small change could be added here. Both options are fine for me.